src/frontend/src/redux/search/reducer.ts

```diff
@@ -40,14 +43,24 @@ export function searchReducer(
       }
       return {...state, questions: questions, loading: true};
     case ANSWER_DISPOSITION:
-      const edits = JSON.parse(JSON.stringify(state.edits));
+      edits = JSON.parse(JSON.stringify(state.edits));
```

Comment:
Why remove the const here? `edits` can just be a different variable with the same name across the two cases.

Reply:
Ah, seems that case blocks don't have a new scope unless I make a new `{}` block. Adding that to allow two `const`s seems fine.